### DISCUSSION TITLE
Access to current user's role and permissions from script

### DISCUSSION THREAD

**Discussion by kpurikhova** [requarks/wiki#5586] [Help / Questions] [answered]
Is the any way to check role of current user from js script integrated to the page properties? It would be very helpful to hide some parts of the page like print button for certain users as example.
![image](https://user-images.githubusercontent.com/96480021/187369289-1d7cabaf-cca1-4b92-b4fe-9b8be77fc780.png)

**Comment by NGPixel** [comment, marked as answer]
You could check the `jwt` cookie, which includes the list of permissions the user has. You simply need to decode it using the [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) library first.
